nest-city-account/src/utils/subservices/__test__/line-logger.subservice.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { LineLoggerSubservice } from './line-logger.subservice'
+import { LineLoggerSubservice } from '../line-logger.subservice'
 
 describe('LineLoggerService', () => {
```

Comment:
Maybe rename this test to `LineLoggerSubservice`

```suggestion
describe('LineLoggerSubservice', () => {
```